app/src/containers/GetUser.js

```diff
@@ -0,0 +1,18 @@
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+import { compose, withProps, toRenderProps } from 'recompose';
+
+const query = gql`
+  query me {
+    me {
+      name
+    }
+  }
+`;
+
+const enhanced = compose(
+  graphql(query),
+  withProps(({ data: { me }}) => me)
+);
+
+export default toRenderProps(enhanced);
```

Comment:
pass user object to nested components, unlike CheckAuth container lets children figure out what is rendered